support_XDP_offload_NIC_names.yaml

```diff
@@ -0,0 +1,3 @@
+# interfaces with offload XDP support
+NIC_names:
+- "Netronome"
```

Comment:
Do you want to make it more specific? Presumably add model names that we support (and tested for)?

Reply:
This program might not work with all Netronome NICs. I think we should restrict it to the models we tested.